otter/convergence/service.py

```diff
@@ -386,6 +386,17 @@ def delete_divergent_flag(tenant_id, group_id, version):
         yield msg('mark-clean-success')
 
 
+def trigger_convergence(tenant_id, group_id):
+    """
+    Trigger convergence on a scaling group
+    """
+    eff = with_log(mark_divergent(tenant_id, group_id),
+                   tenant_id=tenant_id, scaling_group_id=group_id)
+    return eff.on(success=lambda _: msg("mark-dirty-success"),
```

Comment:
Apologies, I probably just don't understand performing `BoundFields` - since "mark-dirty-success" and "mark-dirty-failure" are no longer within the `with_log` context (and hence within `BoundFields`, does this mean that "mark-dirty-success" and "mark-dirty-failure" will not be logged with the `tenant_id` and `scaling_group_id` fields?  Whereas they seem to still be, in `ConvergenceStarter.start_convergence`.